lxd/api.go

```diff
@@ -183,6 +184,11 @@ func restServer(d *Daemon) *http.Server {
 	}
 
 	for _, c := range api10 {
+		// Every 1.0 endpoint should have a type for the API metrics.
+		if !shared.ValueInSlice(c.MetricsType, entity.APIMetricsEntityTypes()) {
+			panic(fmt.Sprintf("Endpoint \"/1.0/%s\" has invalid MetricsType: %s", c.Path, c.MetricsType))
```

Comment:
Please can you use backticks rather than escaping the double quotes, e.g.

`panic(fmt.Sprintf(`Endpoint "/1.0/%s" has invalid MetricsType: %s`, c.Path, c.MetricsType))`

Its easier to read :)